source/slang/slang-session.cpp

```diff
@@ -16,6 +18,31 @@
 namespace Slang
 {
 
+// Helper function to find the neural.slang module path
+static String findNeuralModulePath()
```

Comment:
It feels like this function should be decomposed into `getStandardModulePath` and `findStandardModule(String moduleName)` that does file system lookups in `getStandardModulePath`.